src/components/settingsPanel/SettingsPanel.vue

```diff
@@ -213,6 +213,9 @@ export default {
                 this.$refs.workspaceModal.closeModal();
             }
         },
+        defaultValue(zone) {

```

Comment:
bad name for method, current component contains different types of data and it's not obvious what this method relates to, make it more specific